src/account/UpgradeableModularAccount.sol

```diff
@@ -573,6 +609,41 @@ contract UpgradeableModularAccount is
     // solhint-disable-next-line no-empty-blocks
     function _authorizeUpgrade(address newImplementation) internal override {}
 
+    function _checkIfValidationApplies(bytes4 selector, FunctionReference validationFunction, bool shared)
```

Comment:
This might be a bit outside of the current scope, but do you think we should implement a default validation function that applies to all selectors unless specifically overridden? This could serve as a gatekeeper for the account and potentially lower the installation cost.

Reply:
Yes that's definitely a workflow we want to be able to support: the ability to create a semi-modular account with some builtin features. I think it should probably belong in a new account contract, possibly inheriting from the existing `UpgradeableModularAccount`? But also likely for a different PR.